index.js

```diff
@@ -0,0 +1,90 @@
+import { info, showContact} from "./modules/info.js";
```

Comment:
- [ ] You are importing these files but they are not being used anywhere in your project, please fix this
- [ ] You would also need to  import { DateTime } from "luxon" and use it for your dynamic date and time